tests/queries/0_stateless/01048_exists_query.sql

```diff
@@ -28,7 +28,7 @@ EXISTS t_01048; -- Does not work for temporary tables. Maybe have to fix.
 EXISTS TABLE t_01048;
 EXISTS DICTIONARY t_01048;
 
-CREATE DICTIONARY db_01048.t_01048 (k UInt64, v String) PRIMARY KEY k LAYOUT(FLAT()) SOURCE(HTTP(URL 'http://example.test/' FORMAT TSV)) LIFETIME(1000);
```

Comment:
Had to change `TSV` identifier to `'TSV'` literal, because there were some troubles in parsing and don't know how to fix it better.

Reply:
I will check out of scope of this PR what is going wrong.